store-implementation/map-store/src/main/java/uk/gov/gchq/gaffer/mapstore/impl/MapImpl.java

```diff
@@ -212,6 +222,14 @@ void addIndex(final EdgeSeed edgeSeed, final Element element) {
         edgeIdToElements.put(edgeSeed, element);
     }
 
+    void removeIndex(final EntitySeed entitySeed, final Element element) {
```

Comment:
The method name is similar to the one below could we rename to make it clear what type of index it is removing?